website/content/en/status/report-2023-07-2023-09/mfsbsd-build-integration.adoc

```diff
@@ -0,0 +1,30 @@
+=== Integrate mfsBSD into the release building tools
+
+Links: +
+link:https://wiki.freebsd.org/SummerOfCode2023Projects/IntegrateMfsBSDIntoTheReleaseBuildingTools[Wiki Article] URL: link:https://wiki.freebsd.org/SummerOfCode2023Projects/IntegrateMfsBSDIntoTheReleaseBuildingTools[]
+link:https://reviews.freebsd.org/D41705[Code Review on Phabricator] URL: link:https://reviews.freebsd.org/D41705[]
+link:https://freebsdfoundation.org/blog/meet-the-2023-freebsd-google-summer-of-code-students-soobin-rho/[FreeBSD Foundation Blog Post] URL: link:https://freebsdfoundation.org/blog/meet-the-2023-freebsd-google-summer-of-code-students-soobin-rho/[]
+
+Contact: Soobin Rho <soobinrho@FreeBSD.org>
+
+==== What is mfsBSD?
+
+"mfsBSD is a toolset to create small-sized but full-featured mfsroot based distributions of FreeBSD that store all files in memory (MFS) [Memory File System] and load from hard drive, usb storage device or optical media.
+It can be used for a variety of purposes, including diskless systems, recovery partitions and remotely overwriting other operating systems."
+
+mailto:mm@FreeBSD.org[Martin Matuska] is the creator of mfsBSD, and he is also the author of the original link:https://people.freebsd.org/~mm/mfsbsd/mfsbsd.pdf[mfsBSD white paper], which is where the excerpt above was taken from.
+The upstream mfsBSD is maintained at the link:https://github.com/mmatuska/mfsbsd[GitHub mfsBSD repository].
+
+==== Purpose of this project
+
+This project integrates mfsBSD into the FreeBSD release tool set, creating an additional target of mfsBSD images (.img for disc and .iso for optical images) into the /usr/src/release/Makefile.
```

Comment:
For `.iso`, is _optical images_ the best phrase? 

(I imagine `.iso` files used, maybe more commonly, in other contexts.)

Reply:
I agree! Change commit incoming :+1: